cypress-io/cypress
# feat: Disable file watching in component tests in run mode

- Closes #22959 

### User facing changelog
* Disable file watching when executing component tests in "run" mode

### Additional details
Should have a positive impact on performance by not attempting to watch/reload any dev files in the project when running Cypress tests in `run` mode

### Steps to test
Reproduction instructions are available in #22959. When running with this branch you should not see reload/recompile events if the `App.tsx` file is touched when executing test in `run` mode, but they should still occur in `open` mode

### How has the user experience changed?
No user-facing impact, minor performance gain

### PR Tasks
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
